.github/CODEOWNERS

```diff
@@ -1 +1 @@
-@saradrada @garzuzo @holgiosalos @danielgalvis98 @leonleo997 @manuelq12
\ No newline at end of file
+* @saradrada @garzuzo @holgiosalos @danielgalvis98 @leonleo997 @manuelq12
```

Comment:
EOF = this means that you should add one blank line to the end of the file. That's why GitHub shows this icon:
![image](https://user-images.githubusercontent.com/10209180/126646590-f0e7d891-e2eb-47c1-9b05-4b29a7c66e4a.png)

Reply:
new Revised commit added the line at the end of file in code owners file